notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<p>I feel like we should split up the images for the first motif pair and the second motif pair. Additionally, I think we'd want to explain that not all matrix profile distances are created equal and that, likely, the matrix profile value for the second motif pair is much higher than the matrix profile value for the first motif pair. In other words, there may be no significant relationship between "Dishwasher" and "Freezer" (right??)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>